addons/purchase_stock/wizard/product_replenish.py

```diff
@@ -42,17 +42,6 @@ def _compute_show_vendor(self):
         for rec in self:
             rec.show_vendor = rec._get_show_vendor(rec.route_id)
 
-    @api.onchange('route_id')
-    def _onchange_route_id(self):
```

Comment:
normally we never remove a function from stable in case someone has overridden it (with a customization/3rd party module). Safer would be to update it to only do a `pass` instead

Reply:
Indeed, I first noticed the bug when working in master so that's why I forgot that my fix was only suitable there.
So I guess I should target only the saas-16.4 version for this modification and remove the function in 17 + in forward ports ?